composer.json

```diff
@@ -10,18 +10,18 @@
     }
   ],
   "require": {
-    "php": "^8.1",
-    "behat/behat": "^3.12",
-    "phpunit/phpunit": "^9.5",
-    "symfony/css-selector": "^6.2",
-    "symfony/dom-crawler": "^6.2"
+    "php": "^8.3",
+    "behat/behat": "^3.14",
+    "phpunit/phpunit": "^10.1|^11.0.1",
+    "symfony/css-selector": "^6.2|^7.1",
+    "symfony/dom-crawler": "^6.2|^7.1"
   },
   "require-dev": {
-    "blumilksoftware/codestyle": "^1.10",
-    "illuminate/contracts": "^9.1|^10.0",
-    "illuminate/http": "^9.1|^10.0",
-    "illuminate/support": "^9.1|^10.0",
-    "symfony/http-foundation": "^6.0"
+    "blumilksoftware/codestyle": "^3.1.0",
+    "illuminate/contracts": "^10.0|11.9.2",
+    "illuminate/http": "^10.0|11.9.2",
+    "illuminate/support": "^10.0|11.9.2",
+    "symfony/http-foundation": "^6.2|^7.1"
```

Comment:
I think you can stay with `"^10.0|11.9"` or even `"^10.0|11.0"` if there's no big changes in 11.1.